diff --git a/logger/Logger.swift b/logger/Logger.swift
index 383f477..341360a 100644
--- a/logger/Logger.swift
+++ b/logger/Logger.swift
@@ -31,7 +31,11 @@ private struct TextLog: TextOutputStream {
 
     static var shared = TextLog()
     
+#if DEBUG || ENABLE_LOGGING
+    private var enableLogging = true
+#else
     private var enableLogging = false
+#endif
     
     func write(_ string: String) {
         guard enableLogging else { return print(string) }
